### PR TITLE
Add --skip-documentation option.

### PR DESCRIPTION
**Pull request by jverkoey**
This allows jazzy to skip the documentation generation phase.

Phases I see in Jazzy include:

- Docset generation.
- Documentation generation.

`--skip-documentation` disables both phases.

Another option could be `--skip-docset` which would generate the website documentation, but not the .docset.

Use cases of this flag:

1. As part of a linting engine. Fast feedback is ideal for linting, so skipping documentation generation removes a big chunk of work. See #392 for more details.

### CODE REVIEW

File: lib/jazzy/doc_builder.rb [hunk 1]
```diff
@@ -121,16 +118,23 @@ def self.build_docs_for_sourcekitten_output(sourcekitten_output, options)
       end
 
       source_module = SourceModule.new(options, docs, structure, coverage)
-      build_docs(output_dir, source_module.docs, source_module)
 
-      write_undocumented_file(undocumented, output_dir)
+      if !options.skip_documentation
```
Comment (segiddins) [inline lib/jazzy/doc_builder.rb]: `unless`

Comment (jverkoey) [inline lib/jazzy/doc_builder.rb]: Fixed!

**Comment by jverkoey** [comment]
Hrm, there's no easy way that I see to update the integration specs without having push access to the primary repo. Do I have to land the other pull request first?

https://github.com/realm/jazzy-integration-specs/pull/8

**Comment by jverkoey** [comment]
This pull request is now also green, w00t! @jpsim leaving up to y'all :)

**Comment by jpsim** [comment]
Looks good! :+1:

**Pull request merged**